### PR TITLE
chore: drop Node.js 8, 10 & update version from 1.0.0 to 2.0.0

### LINKED ISSUE #8: update pug to 3.0.2
hope update pug to 3.0.2

### PR DESCRIPTION
**Pull request by yoshinorin**
As is the title.
I will publish `2.0.0` after merge this PR.

Thank you :)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,6 +38,6 @@
     "nyc": "^15.1.0"
   },
   "engines": {
-    "node": ">=8.6.0"
+    "node": ">=10.13.0"
```
Comment (JLHwung) [inline package.json]: Node 10 reaches EOL at Apr 30. Consider bump to `>=12.17.0`, which has native ESM support.

Comment (yoshinorin) [reply package.json]: Thank you :)

Yes, I know. But, the current [hexojs/hexo](https://github.com/hexojs/hexo) is not yet dropped v10. 
IMHO we should not bump to `12.x` until [hexojs/hexo](https://github.com/hexojs/hexo) drops `v10` 😓 

> https://github.com/hexojs/hexo/blob/eef2eac874b5118cae73fcdb4acba3a6c744f2b5/package.json#L80

Comment (stevenjoezhang) [reply package.json]: `hexo-util` now requires Node.js 12.4.0 https://github.com/hexojs/hexo-util/blob/576fdd6a0c8b16be4cb63f1c1f38ee1f9508dc24/package.json#L56

Comment (yoshinorin) [reply package.json]: @stevenjoezhang 
Done :)

**Review by stevenjoezhang — APPROVED** [review]

**Review by tomap — APPROVED** [review]

**Comment by yoshinorin** [comment]
@curbengh @tommy351 @NoahDragon
We want to publish a new version.
Would you please publish a new version or permit it to me at npm?

**Comment by tommy351** [comment]
@yoshinorin I just added `hexo-renderer-pug` package to the `developers` team of `hexojs` group on npm. I am not sure if it works or not. Please try publishing again.

**Comment by yoshinorin** [comment]
@tommy351 
Thank you ❤️ 
I just published `v2.0.0`

I will write a release note :)

**Pull request merged**